tests/snmpsim/netmodule.snmprec

```diff
@@ -0,0 +1,2 @@
+1.3.6.1.2.1.1.1.0|4|NB1600
```

Comment:
Is this the full sysDescr?

Reply:
This is just from what was in the snmpwalk in issue: https://github.com/librenms/librenms/issues/7390

Reply:
Ace. Thanks.